crates/zoon/src/element/button.rs

```diff
@@ -22,16 +99,30 @@ impl Button<LabelFlagNotSet, OnPressFlagNotSet> {
                         .style("margin-top", "auto")
                         .style("margin-bottom", "auto"),
                 )
-                .style_group(StyleGroup::new(".button > .align_top").style("margin-bottom", "auto"))
-                .style_group(StyleGroup::new(".button > .align_bottom").style("margin-top", "auto"))
                 .style_group(
-                    StyleGroup::new(".button > .align_left").style("align-self", "flex-start"),
+                    StyleGroup::new(".button > .align_top")
```

Comment:
Intelliji does not make in reformatting outside of rustfmt.toml. In `Seed` we have the `rustfmt.toml`. I stole the one we used in there and stash it in my MZ to use it because we do not have any. I run on nightly. 

Here is my rustfmt.toml

```toml

# https://github.com/rust-lang/rustfmt/blob/master/Configurations.md

newline_style = "Unix"
use_field_init_shorthand = true
use_try_shorthand = true
#
#
## nightly
#
normalize_doc_attributes = true
#reorder_impl_items = true
normalize_comments = true
format_code_in_doc_comments = true
wrap_comments = true
#wrap_comments = true
#format_strings = true
#match_block_trailing_comma = true
#trailing_comma = "Always"
#merge_imports = true
#error_on_line_overflow = true
#error_on_unformatted = true
#report_todo = "Always"
#report_fixme = "Always"
#version = "Two"
max_width=100

```
Yes my formatting is polluting this PR and I am sorry for that. I try to track as much as I can but sometimes I do not see them because my PR is too bog lol. 
From the rustfmtl.toml file I have, it seems some formatting was forgotten at least according to the rules I used.
Jetbrain run `cargo fmt` on every auto save every time I change some code.

Reply:
Please use stable Rust.
Please format it with the default formating settings - I hope `cargo fmt --all` + "unwraping" comments to the previous form could be enough.
But don't kill too much time with it 🙂

Reply:
Okay, I will stay in stable for future :wink: